docs/src/_reference/command-line-interface.md

```diff
@@ -1279,6 +1279,41 @@ echo '{"singer_state": {"project_123456_issues": "2020-01-01"}}' > gitlab_state.
 meltano state set --force dev:tap-gitlab-to-target-jsonl --input-file gitlab_state.json
 ```
 
+### vacuum
```

Comment:
My take on this: the `state` command verbs align with the perspective of state management, while the previous runs added to the table is the implementation detailed abstracted away from that interface. If we say `state delete`, that could easily gets confused with the semantic of `state clear`. I agree with your point on `vacuum` being specific to database. However it also has the upside of stating 1. it reduce disk space usage for housekeeping purpose, and 2. it does not change the data state. That's the reason behind my choice of term `vacuum`, but I am open to any other options as long as you think it best serves the purpose.

Reply:
This is great @simonpai 👍 I agree that `vacuum` is a little too database-specific, and maybe even misleading in the database context (generally the VACUUM operation reorders data, rather than deleting it, akin to defragmentation for disk drives). The docker CLI uses `prune` for its clear-up operations (deleting unused images/containers/volumes etc.) which might work for us too; it meets your 2 points. Also, many users will be familiar with both tools, so some level of consistency is advantageous. As always, naming things is the hardest part 😅

Reply:
`prune` sounds like the best candidate so far! I think I'll go with `meltano state prune`. I appreciate feedback from both of you. TBH, it took me a while to think about this and I'm glad to find a better answer from discussion with you guys.